### ISSUE TITLE
Replacing regular/bold with other weights when building custom blend of Iosevka?

### ISSUE THREAD

**Issue by oxysoft** [be5invis/Iosevka#276]
I have built my own custom blend of Iosevka with the variants I like, named `Iosevka-Oxysoft`. I use the Extralight weight in my IDE, (displayed as `Iosevka-Oxysoft Extralight`) however bold only works when I pick the regular font (`Iosevka-Oxysoft`). 

Is there a way to build Iosevka without generating all the different weight, keeping only the "main font" (`Iosevka-Oxysoft` in my case)  and instead, manually specifying which weight to use for normal and bold? Or perhaps there is another solution that would simply make bold work for extralight?

Thanks!

**Comment by lshoravi** [comment]
The way I fool emacs is to move the fonts I don't want put of my font directory ´´~/.fonts/´´ (in this case I want medium to be bold) and renaming them accordingly, i.e

´´iosevka-X-bold -> somewhere else (trash)´´
´´ iosevka-X-medium -> iosevka-X-bold´´

**Comment by oxysoft** [comment]
Hm I don't think this will be enough to fool my Rider IDE, I don't think it goes off of filenames like that to determine was is regular, bold, etc. From my understanding, it seems to use the actual font name within embedded within the metadatas rather than the filename. Still, I'll give it a shot!

**Comment by lshoravi** [comment]
Emacs also goes by metadata but falls back to something if its missing, I
figured maybe yours does the same :D

**Comment by oxysoft** [comment]
As I had expected, this IDE is a little smarter, file names don't matter. Hopefully there is some new build options to just build a regular font where the weights are hand-picked. 

e.g.
```
Regular              ->  Extralight
Regular bold         ->  Medium
Regular italic       ->  Extralight Italic
Regular italic bold  ->  Extralight Medium Italic
```

**Comment by oxysoft** [comment]
Thank you for redoing the building process (**much nicer**) but this actually didn't fix it! HOWEVER I did find an alternative route to fix my problem. If I change regular to `200` it is still registered as "Extralight" for the font's internal name which my IDE doesn't like displaying bold for. What I did is I went into parameters.toml and then copied the values for `w-200` into `w-400`, and then `w-500` into `w-700` for bold.
